src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/extract_ability.json

```
@@ -0,0 +1,13 @@
+{
+  "title": "Extract Ability",
+  "image": {
+    "file": "tconstruct:book/images/worktable/extract_ability.png",
+    "width": 147,
+    "height": 64
+  },
+  "text": [
+    {
+      "text": "Extracting ability modifiers from enchanted tools is a quite expensive process, requiring not only ichor crystals but also a diamond to help the process. Even with the power of ichor, the tool is still lost, seems you need enderslime if you wish to preserve the tool."
```

```suggestion
  "title": "能力强化提取",
  "image": {
    "file": "tconstruct:book/images/worktable/extract_ability.png",
    "width": 147,
    "height": 64
  },
  "text": [
    {
      "text": "从附魔工具上提取能力强化的过程相当费材料，不止需要灵浆水晶，还需要钻石来辅助提取。就算有灵浆的力量支持，工具还是没有办法保留下来，似乎只有末影黏液才能让你保住工具了。"
```
may have many colloquial word choices, do accept suggestion with caution